changelog/pending/20250805--cli-plugin--add-list-alias-to-pulumi-policy-ls.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli/plugin
+  description: Add "list" alias to `pulumi policy ls`.
```

```suggestion
  description: Add "list" alias to `pulumi policy ls`
```

Changelog entries do not have a final period.